app/Http/helpers.php

```diff
@@ -71,3 +71,22 @@ function successResponse($message = '', $data = '', $statusCode = 200)
 
     return response()->json($response, $statusCode);
 }
+
+
+/**
+ * Gets time in logged in user's timezone
+ * @param string $dateTimeString
+ * @param string $format
+ * @return string
+ * @throws Exception
+ */
+function getTimeInLoggedInUserTimeZone(string $dateTimeString, $format = 'M j, Y, g:i a')
+{
+    // caching for 4 seconds so for consecutive queries, it will be readily available. And even if someone updates their
+    // timezone, it will start showing the new timezone after 4 seconds
+    $timezone = Cache::remember('timezone_'.Auth::user()->id, 5, function(){
+        return isset(Auth::user()->timezone) ? Auth::user()->timezone->name : 'UTC';
```

Comment:
Do we need this check here? We can directly get the timezone name since the timezone saved is never null

Reply:
Just for the fallback. I wasn't confident that it can't be null, so added. Are we confident?

Reply:
Yes, even in the database directly, it cannot be set as null, unless ofcourse someone forcibly tries to do it.

Reply:
removed the check